vllm/core/block/prefix_caching_block.py

```diff
@@ -54,6 +53,8 @@ def __init__(
 
         self._block_size = block_size
 
+        self.evictor: Evictor = make_evictor(eviction_policy)
```

Comment:
nit: add some comment explaining the role

Reply:
Got it~